cdap-distributions/src/COPYRIGHT/postcss-value-parser@4.1.0/COPYRIGHT

```diff
@@ -0,0 +1,22 @@
+Copyright (c) Bogdan Chadkin <trysound@yandex.ru>
+
+Permission is hereby granted, free of charge, to any person
```

Comment:
Same here. It does not look like a standard license.

Reply:
Is it MIT?

Reply:
Here it mentions it's MIT: https://www.npmjs.com/package/postcss-value-parser